collect_app/src/androidTest/java/org/odk/collect/android/support/pages/UserInterfacePage.java

```diff
@@ -22,8 +26,15 @@ public UserInterfacePage clickOnLanguage() {
     }
 
     public MainMenuPage clickOnSelectedLanguage(String language) {
-        onView(withText(language)).perform(ViewActions.scrollTo());
-        onView(withText(language)).perform(click());
+        try {
+            onView(withText(language)).perform(click());
+        } catch (NoMatchingViewException e) {
+            for (int i = 0; i < 10; i++) {
```

Comment:
Scrolling didin't work in that list of languages so I used this nasty workaround.